fastlane/Fastfile

```diff
@@ -25,6 +25,15 @@ USER_ENV_FILE_PATH = File.join(Dir.home, '.a8c-apps', ENV_FILE_NAME)
 
 TEAM_ID_APP_STORE = 'PZYM8XX95Q'
 TEAM_ID_ENTERPRISE = '99KV9Z6BKV'
+EXTERNAL_BETA_TESTER_GROUPS = [
+  'A8cs',
+  'Alpha Slack',
+  'Beta Slack',
+  'Stable Slack',
+  'Reddit',
+  'Big Databases',
```

Comment:
```suggestion
  'Big Databases',
  'Reddit',
  'Stable Slack',
```

Just alphabetizing 🙂

Reply:
Good point… though that means that `* Slack` groups are not together anymore now 😄 

That being said, the groups are also listed alphabetically in App Store Connect, so maybe that still make sense to alphabetize them here too after all 🙂